### PR TITLE
Formatted JSON code

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
## Purpose of this PR

Properly formatted the JSON code of `auth.json` file.

## Affected URLs
- https://devdocs.magento.com/guides/v2.3/cloud/trouble/troubleshoot-deployment.html

## Links to source code
- https://github.com/magento/magento2/blob/2.3-develop/auth.json.sample

screenshot

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/cloud/trouble/troubleshoot-deployment.md [hunk 1]
```diff
@@ -32,22 +32,25 @@ To redeploy with the correct authentication keys:
...
 
-    {
-       "http-basic": {
-          "repo.magento.com": {
-             "username": "<your public key>",
-             "password": "<your private key>"
-          }
-       }
-    }
+```json
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/trouble/troubleshoot-deployment.md]: This code block must be indented 4 spaces.

File: guides/v2.1/cloud/trouble/troubleshoot-deployment.md [hunk 2]
```diff
@@ -32,22 +32,25 @@ To redeploy with the correct authentication keys:
...
+```
 
 6.	Add the correct authentication keys.
 7.	Save your changes and exit the text editor.
 8.	Commit and merge your changes.
-
-		git add -A
-		git commit -m "<description of change>"
-		git push origin master
+```bash
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/trouble/troubleshoot-deployment.md]: This code block must be indented 4 spaces.

File: guides/v2.1/cloud/trouble/troubleshoot-deployment.md (hunk 1 above)
Comment (yogeshsuhagiya) [reply guides/v2.1/cloud/trouble/troubleshoot-deployment.md]: @jeff-matthews  Added 4 spaces

File: guides/v2.1/cloud/trouble/troubleshoot-deployment.md (hunk 2 above)
Comment (yogeshsuhagiya) [reply guides/v2.1/cloud/trouble/troubleshoot-deployment.md]: @jeff-matthews Added 4 spaces

**Pull request merged**